src/app/queries/hooks.ts

```diff
@@ -38,7 +38,7 @@ export const playbookSchema = yup
   .test('valid-yaml', 'Playbook must be valid YAML', (value, context) => {
     try {
       yaml.load(value || '');
-    } catch (e) {
+    } catch (e: any) {
```

Comment:
I was about to say we can use `YAMLException` from `js-yaml` as the type here, but disappointingly they use `any` for the types of `reason` and `mark` anyway. So not much would be gained there.

Reply:
Ah good call - we can make that change in a followup PR.. every little bit helps I suppose :)